borg/_hashindex.c

```diff
@@ -13,10 +13,12 @@
 #endif
 
 #if (defined(BYTE_ORDER)&&(BYTE_ORDER == BIG_ENDIAN)) ||  \
+    (defined(__BYTE_ORDER__) && (__BYTE_ORDER__ == __ORDER_BIG_ENDIAN__)) || \
```

Comment:
keep the tight style (no blanks around &&) and do the style change globally later?